nncf/pruning/filter_pruning/algo.py

```diff
@@ -111,11 +115,12 @@ def get_in_out_channels(module):
         graph = self._model.get_original_graph()
         for nncf_node in graph.get_all_nodes():
             node_module = self._model.get_module_by_scope(nncf_node.op_exec_context.scope_in_model)
+            scope = nncf_node.op_exec_context.scope_in_model
             in_channels, out_channels = get_in_out_channels(node_module)
             if in_channels:
-                self.modules_in_channels[nncf_node.node_id] = in_channels
+                self.modules_in_channels[scope] = in_channels
```

Comment:
@mkaglins I had to refactor some of your code to use module scopes instead of NNCF graph node keys when referring to prunable modules. The tests are passing, but still please review.

Reply:
@vshampor Looks good.